src/pages/commits/index.tsx

```diff
@@ -87,6 +91,10 @@ const Commits = () => {
   const statusFilters = toArray(parsed[ProjectFilterOptions.Status]);
   const variantFilters = toArray(parsed[ProjectFilterOptions.BuildVariant]);
   const taskFilters = toArray(parsed[ProjectFilterOptions.Task]);
+  const [viewFilter] = useQueryParam(
+    ProjectFilterOptions.View,
+    "" as ProjectHealthView
+  );
```

Comment:
I'm okay with this but I just had a follow-up question of if the URL and app state need to be the same for the default case -- for example, `chartOpen` and `chartType` default to a specific value but aren't included in the URL if the default is applied. Should we update the behavior of those URL params?

Reply:
I guess my thinking is it doesn't matter as much in a case where we have a predictable default, i.e. there the chart always defaults to open and we'll use a query param if it should be closed. But since the view default state depends on a project setting it would've been harder to deal with an indeterminate query param. Also would be interested in any of your thoughts @khelif96!

Reply:
In general, it makes sense to provide default values for stateful values.  
We have set a precedent for this type of stateful behavior in parsley. Several log viewing settings have default values despite not being saved to the URL initially.

I do agree with Minna's change however we should be setting a default value. It provides us with a fallback in case the graphql query fails and it also provides us with type safety. (Right now `ViewFilter` is typed as `ProjectHealthView` but is actually an empty string because of the type assertion)

Reply:
Spoke about this offline. This use case makes sense but with some slightly different typing to promote better type safety.